python/doc/examples/data_analysis/distribution_fitting/plot_model_singular_multivariate_distribution.py

```diff
@@ -41,29 +41,76 @@ def draw(dist, Y):
 
 
 # %%
-# Generate some multivariate data to estimate, with correlation.
+# We consider the function :math:`f: \Rset^3 \rightarrow \Rset` defined by:
+#
+# .. math::
+#
+#      f(u, v_1, v_2) = (y_1, y_2)
+#
+# where:
+#
+# .. math::
+#
+#      y_1 & = sin(u) / (1 + cos(u)^2) + 0.05 * v_1 \\
```

Comment:
Use \sin and \cos instead of bare sin and cos, and remove the multiplication sign

Reply:
yes